data/docs/userguide/fluentd_to_signoz.mdx

```diff
@@ -109,7 +109,7 @@ At SigNoz we use opentelemetry collector to recieve logs which supports the flue
     ```
     Here we are adding our clickhouse exporter and creating a pipeline which will collect logs from `fluentforward` receiver, processing it using batch processor and export it to clickhouse.
 
-* Expose the port in port for otel-collector in `docker-compose.yaml` file present in `deploy/docker/clickhouse-setup`
+* Expose the port in port for otel-collector in `otel-collector-config.yaml` file present in `deploy/docker/clickhouse-setup`
```

Comment:
this can be changed to `docker-compose-core.yaml` and not otel collector config

Reply:
let me check.

Reply:
Correct, Let me correct it